src/ice/agent.rs

```diff
@@ -316,6 +320,39 @@ impl IceAgent {
         }
     }
 
+    /// Sets the initial STUN **R**etransmission **T**ime**O**ut.
```

Comment:
Nice one :) 

My first reaction was WTF. l33T 5p3ak?!

Reply:
It looks better in the rendered docs :D